lib/broadside/version.rb

```diff
@@ -1,3 +1,3 @@
 module Broadside
-  VERSION = '2.0.0'
+  VERSION = '3.0.0-prerelease'
```

Comment:
@matl33t i noticed that some other more mature projects use this approach to versioning - what's on master is the next tag + `-prerelease`.  I think that's a good strategy; way better than the two other options at least.